Sorting Algorithms/Quick Sort Algorithm/Java/QuickSortRandomPivot.java

```diff
@@ -0,0 +1,95 @@
+
+/* ===== ===== =====
+
+Theory of Programming
+
+Quick sort algorithm with random pivot selection
+http://theoryofprogramming.com/2015/01/20/quick-sort-algorithm/
+GitHub - https://github.com/VamsiSangam/theoryofprogramming
+Code Contributor - Aditya Aggarwal (github.com/the-cybersapien, cybersapien.xyz)
+
+===== ===== ===== */
+
+import java.util.Scanner;
+
+
+public class Solution {
```

Comment:
Sorry, errors like these happen when one uses an IDE to code and then use a text editor to add text and create a file :laughing: 
Fixed it. Please review.